cmd/diff.go

```diff
@@ -116,6 +116,14 @@ func diffImages(image1Arg, image2Arg string, diffArgs []string) error {
 	return nil
 }
 
+func processImageName(imageName string) string {
+	if pkgutil.IsTar(imageName) || strings.HasPrefix(imageName, "daemon://") {
```

Comment:
What if it already has remote://? would this add it a second time?

Reply:
I wasn't really planning on advertising that prefix to users, but I guess there's nothing stopping them from adding it in which case this would break. fixed